ldconfig

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+ldconfig
```

Comment:
Could you move this to the cmake folder and add a short note for what it's used for?